cinema/serializers.py

```diff
@@ -1 +1,119 @@
-# write serializers here
+from rest_framework import serializers
+from rest_framework.fields import SerializerMethodField, IntegerField
+from rest_framework.relations import SlugRelatedField
+
+from cinema.models import (
+    Genre,
+    Actor,
+    Movie,
+    CinemaHall,
+    MovieSession,
+    Order,
+    Ticket
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+        read_only_fields = ["id"]
+
```

Comment:
no need to set id field read only. this is done automatically. check other places